### PR TITLE
feat: add Supporters stat to project profile (closes #842)

### LINKED ISSUE #842: Add "Supporters" profile stat to project profile
image

### PR DESCRIPTION
**Pull request by evvvritt**
closes #842 
- side scroll on mobile

(ps @efstajas - is it a memory/performance concern that the Pile.svelte component currently takes in a larger list of components than are displayed?)

Screenshot 2023-12-22 at 13 58 47
Screenshot 2023-12-22 at 13 59 07

### CODE REVIEW

**Review by efstajas — CHANGES_REQUESTED** [review]
Looks good but I think it should either use `drip-list-badge` with `showName` set to `true`, or replace the avatar part in that component with `drip-list-avatar`. Right now it's duplicated in those two places.

**Comment by efstajas** [comment]
@brandonhaslegs you can always use the PR preview deploys. They're connected to Goerli.

**Comment by brandonhaslegs** [comment]
@efstajas Yeah i know, I just need a link to a project that is on Goerli.

**Comment by evvvritt** [comment]
@efstajas I now use DL-avatar inside DL-badge. 

@brandonhaslegs I can't find a DL supporting a project on goerli, you could pull this branch locally and set your local app to run mainnet(?)

File: src/lib/components/unclaimed-project-card/unclaimed-project-card.svelte [hunk 1]
```diff
@@ -53,13 +53,15 @@
...
-  }));
+  $: unclaimedTokenPile = mergedUnclaimedFunds
+    ?.map((fund) => ({
+      component: Token,
+      props: {
+        address: fund.tokenAddress,
+        show: 'none',
+      },
+    }))
+    .slice(0);
```
Comment (efstajas) [inline src/lib/components/unclaimed-project-card/unclaimed-project-card.svelte:64]: what's the point of `slice(0)` here? 🤔

**Review by efstajas — APPROVED** [review]
LGTM 👍

File: src/lib/components/unclaimed-project-card/unclaimed-project-card.svelte (hunk 1 above)
Comment (evvvritt) [reply src/lib/components/unclaimed-project-card/unclaimed-project-card.svelte]: oops, leftover from some previous ideas. removing

**Pull request merged**